tests/compile-fail/unsupported_foreign_function.rs

```diff
@@ -4,6 +4,6 @@ fn main() {
     }
 
     unsafe {
-        foo(); //~ ERROR unsupported operation: can't call foreign function: foo
+        foo(); //~ ERROR unsupported Miri functionality: can't call foreign function "foo"
```

Comment:
Why'd you edit this test? Behavior without the new flag should not change, should it?

Reply:
I misinterpreted your previous comment on making the error messages the same. Will fix in the next push.